packages/xstate-inspect/test/inspect.test.ts

```diff
@@ -340,4 +340,40 @@ describe('@xstate/inspect', () => {
         .filter((message: any) => message.type === 'service.event')
     ).toHaveLength(1);
   });
+
+  describe('browser inspector', () => {
+    const windowMock = jest.fn() as unknown as Window;
+    let windowSpy;
+
+    beforeEach(() => {
+      windowSpy = jest.spyOn(window, 'open');
+      windowSpy.mockImplementation(() => windowMock);
+    });
+
+    afterEach(() => {
+      windowSpy.mockRestore();
+    });
```

Comment:
similarly, i would move this stuff to the tests that are using it - we can wrap repetitive setup in a helper but i find it much easier to reason about when the whole implementation of the test is explicitly stated in the test body itself. Some truly global teardowns can sometimes be put in `afterEach` and stuff like that (I'm not rly dogmatic about it) but mocks, spies etc are (IMHO) better to be initialized in the test themselves

Reply:
Sounds good to me. I'll make this change as well.